EDA/EDA.py

```diff
@@ -0,0 +1,272 @@
+"""sub011, task001_run_001"""
+# library
+import nibabel as nib
+import numpy as np 
+import diagnostics as diag
+import matplotlib.pyplot as plt
+import math
+
+#functions
+def events2neural(task_fname, tr, n_trs):
+    """ Return predicted neural time course from event file `task_fname`
+
```

Comment:
You can make a comment here about the assumption we make when the TRs do not line up exactly with intervals in the event file.